### ISSUE TITLE
SR-IOV Support for Intel Arc A380 on AMD Systems with i915-sriov-dkms?

### ISSUE THREAD

**Issue by 10bn** [strongtz/i915-sriov-dkms#148]
I'm interested in using the i915-sriov-dkms module for SR-IOV with an Intel Arc A380 GPU on an AMD system. 

Does anyone know if this is generally possible at this stage? 

Thanks for your work on this project!

**Comment by michael-pptf** [comment]
A series does not support SRIOV.

**Comment by nikolaisen-hhpv** [comment]
There're rumours you could unlock it for the A770. Any updates on that matter?

**Comment by michael-pptf** [comment]
> There're rumours you could unlock it for the A770. Any updates on that matter? Reference: LVL1 Tech Forum / Youtube Videos

Nope. Wendell only stated he had one A770 working by cross-flashing to DG2 datacenter CPU, and only that particular model of A770 can be flashed. He did not mention which model it is and the process to perform the cross-flash. 

You can safely treat this as a deader-than-dead-end.
